docs/userGuide/tweakingThePageStructure.md

```diff
@@ -239,6 +240,38 @@ A siteNav has a fixed width of 300 pixels for its contents. A siteNavs is [_resp
 There is no limit to the number of nesting levels or the number of items in the menu, but note that any content exceeding a height of 1000 pixels will be cut off.
 
 
+<hr><!-- ======================================================================================================= -->
+
+## Page Navigation Menus
+
+**You can also add a <trigger trigger="click" for="modal:pageStructure-pageNavidationMenu">_page navigation menu_</trigger> to a page.**
+
+<modal title="Site Naviation Menu" id="modal:pageStructure-pageNavidationMenu">
+  <include src="glossary.md#page-navigation-menu" />
+</modal>
+
+Steps to add a page navigation menu ==(_pageNav_ for short)==:
+1. Specify your pageNav within the `<frontmatter>` of a page with <tooltip content="The value `default` will use `headingIndexingLevel` within `site.json`.">`"default"`</tooltip> or a <tooltip content="HTML defines six levels of headings, numbered from <br>`1 to 6`.">`heading level`</tooltip>.
+2. `[Optional]` You may also specify page navigation header text within `<frontmatter>` that will be placed at the top of the page navigation menu.
```

Comment:
* We currently use "(Optional)" instead of \`[Optional]\` in our documentation.
* Would it be better to say "page navigation title" instead of "header text"?